### PR TITLE
Update sdk

### PR DESCRIPTION
**Pull request by achilleas-kal**

### CODE REVIEW

File: pyinjective/async_client.py [hunk 1]
```diff
@@ -73,6 +78,8 @@ def __init__(
         self.stubInsurance = insurance_rpc_grpc.InjectiveInsuranceRPCStub(self.exchange_channel)
         self.stubSpotExchange = spot_exchange_rpc_grpc.InjectiveSpotExchangeRPCStub(self.exchange_channel)
         self.stubDerivativeExchange = derivative_exchange_rpc_grpc.InjectiveDerivativeExchangeRPCStub(self.exchange_channel)
+        self.stubExplorer = explorer_rpc_grpc.InjectiveExplorerRPCStub(exchange_channel)
+        self.stubAuction = auction_rpc_grpc.InjectiveAuctionRPCStub(exchange_channel)
```
Comment (DrunkRandomWalker) [inline pyinjective/async_client.py:82]: should be self.exchange_channel

File: pyinjective/composer.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 from .proto.injective.exchange.v1beta1 import exchange_pb2 as injective_exchange_pb
 from .proto.injective.types.v1beta1 import tx_response_pb2 as tx_response_pb
 
+from .proto.injective.auction.v1beta1 import tx_pb2 as injective_auction_tx_pb
```
Comment (DrunkRandomWalker) [inline pyinjective/composer.py:8]: This is not used

Comment (achilleas-kal) [reply pyinjective/composer.py:8]: Yeah it will be when we add bids.

File: pyinjective/async_client.py (hunk 1 above)
Comment (achilleas-kal) [reply pyinjective/async_client.py:82]: Cool, I'll change didn't notice the refactoring.

**Pull request merged**